src/Toast/Toast.tsx

```diff
@@ -1,13 +1,24 @@
 import React from "react";
 import { CSSTransition } from "react-transition-group";
+// @ts-ignore
```

Comment:
Ignoring ts warnings or errors doesn't give us much confidence in our code, does it?

Reply:
(alias) class CSSTransition<Ref extends HTMLElement | undefined>
import CSSTransition
'CSSTransition' cannot be used as a JSX component.
  Its instance type 'CSSTransition<HTMLElement | undefined>' is not a valid JSX element.
    The types returned by 'render()' are incompatible between these types.
      Type 'import("/Users/stavio/Documents/Projects/Production/react-flash-message/node_modules/@types/testing-library__react/node_modules/@types/react/index").ReactNode' is not assignable to type 'React.ReactNode'.
        Type '{}' is not assignable to type 'ReactNode'.ts
        
        
        
        this is the error i am getting dont seem to find a solution to it